### PR TITLE
Coloured geras

### PR DESCRIPTION
**Pull request by Doctor-Cpu**
## About the PR

Geras are now coloured the same as the base slime.

## Why / Balance

All geras were the same colour despite them just being reformed slimes.

## Technical details

Gets the slimes base colour and sets the geras colour to said value.

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

:cl:
- add: Geras are now coloured.

### CODE REVIEW

File: Content.Client/Geras/GerasSystem.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            return;
+        }
+        if (TryComp<SpriteComponent>(childUid, out var sprite))
+        {
+            skinColor = appearanceComp.SkinColor;
+            sprite.Color = skinColor;
+        }
+        return;
+    }
+}
```
Comment (mirrorcult) [inline Content.Client/Geras/GerasSystem.cs:34]: do not use network events for this, use `AppearanceComponent`

https://docs.spacestation14.com/en/ss14-by-example/making-a-sprite-dynamic.html

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by Just-a-Unity-Dev** [comment]
cute

File: Content.Client/Geras/GerasSystem.cs (hunk 1 above)
Comment (Doctor-Cpu) [reply Content.Client/Geras/GerasSystem.cs]: I think I've done it right now?

**Comment by Emisse** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Comment by Doctor-Cpu** [comment]
Judging by all the new prs I'm guessing freeze is over.
Could this get reopened?

**Comment by Doctor-Cpu** [comment]
image
Made the geras less transparent.

File: Content.Client/Geras/GerasSystem.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using Content.Client.Geras.Component;
+using Content.Shared.Geras;
+using Robust.Client.GameObjects;
+using static Robust.Client.GameObjects.SpriteComponent;
+
+namespace Content.Client.Geras;
+
+public sealed class GerasSystem : VisualizerSystem<GerasComponent>
```
Comment (AJCM-git) [inline Content.Client/Geras/GerasSystem.cs:8]: use a generic visualizer (we have docs about them in docs.ss14.io)

File: Content.Client/Geras/GerasComponent.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+using Content.Shared.Geras;
+
+namespace Content.Client.Geras.Component;
+
+[RegisterComponent]
+public sealed partial class GerasComponent : SharedGerasComponent
```
Comment (AJCM-git) [inline Content.Client/Geras/GerasComponent.cs:6]: remove this after you do that

**Review by AJCM-git — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
i think dragon random colour could be reused as is with similar code to set the geras colour from the slime

File: Content.Client/Geras/GerasSystem.cs (hunk 2 above)
Comment (AJCM-git) [reply Content.Client/Geras/GerasSystem.cs:8]: or use the dragon random colour like deltanedas said

**Comment by TheShuEd** [comment]
Given that geras has been removed from the game, this no longer makes sense. Thanks for the contribution!

**Pull request closed without merging**